projects/batfish-client/src/main/java/org/batfish/client/BfCoordWorkHelper.java

```diff
@@ -477,6 +478,42 @@ public Container getContainer(String containerName) {
     }
   }
 
+  /**
+   * Returns the questions configured in the remote coordinator service
+   *
+   * @return {@link Map containing Question Names and Question JSON strings}
+   */
+  @Nullable
+  public Map<String, String> getGlobalQuestions() {
+    try {
```

Comment:
Would it make sense to rename this function getQuestionTemplates() to reflect the end point its calling?

Reply:
Now using the original getQuestionTemplate client function, dropped getGlobalQuestions()